presto-verifier/src/main/java/com/facebook/presto/verifier/prestoaction/PrestoExceptionClassifier.java

```diff
@@ -179,4 +168,34 @@ private static boolean isTargetTableAlreadyExistsException(PrestoQueryException
                 && queryException.getQueryStage().isSetup()
                 && TABLE_ALREADY_EXISTS_PATTERN.matcher(queryException.getMessage()).find();
     }
+
+    public static class Builder
+    {
+        private final ImmutableSet.Builder<ErrorCodeSupplier> recognizedErrors = ImmutableSet.builder();
+        private final ImmutableSet.Builder<ErrorCodeSupplier> retryableErrors = ImmutableSet.builder();
+
+        private Builder()
+        {
+        }
+
+        public Builder addRecognizedErrors(Iterable<ErrorCodeSupplier> errors)
```

Comment:
They are also used in PrestoFacebookExceptionClassifier in presto-facebook/presto-facebook-verifier. Verifier is designed in a way so that components can be extended.